generators/app/templates/src/redux/middlewares/analyticsMiddleware.ts

```diff
@@ -2,11 +2,11 @@ import { NavigationActions, StackActions } from 'react-navigation';
 import analytics from '@react-native-firebase/analytics';
 import { getPreviousRouteName } from '@utils/navUtils';
 
-const trackNavigation = routeName => {
-  if (routeName) analytics().setCurrentScreen(routeName, routeName);
+const trackNavigation = (routeName: string) => {
+  if (routeName) analytics().setCurrentScreen(routeName, `${routeName}.js`);
 };
 
-const eventsTrackingMiddleware = ({ getState }) => next => action => {
+const eventsTrackingMiddleware = ({ getState }: any) => (next: any) => (action: any) => {
```

Comment:
Type this with the `State` we have in interfaces, next with `Dispatch` and action with the action you've defined

Reply:
Done